Config.properties

```diff
@@ -12,4 +12,4 @@ implicitTime = 20
 explicitTime = 30
 sleepTime = 500
 language = en
-testEnvironment = dev02
\ No newline at end of file
+testEnvironment = dev03
```

Comment:
add gitignore this file

Reply:
We need the config properties, beacuse the login information now is encrypted